<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamConfigPage</class>
 <widget class="QWidget" name="ExamConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_ExamConfig">
   <item>
    <widget class="QScrollArea" name="scrollArea_ExamConfig">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent_ExamConfig">
      <layout class="QVBoxLayout" name="verticalLayout_ExamConfigInner">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_EC_Header">
         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="text">
            <string>&lt; ËøîÂõû</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_ExamConfigTitle">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Ê®°ÊãüËÄÉËØïÈÖçÁΩÆ</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_EC_Header">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_EC_ConfigSelect">
         <item>
          <widget class="QLabel" name="lbl_ConfigSelect">
           <property name="text">
            <string>ÈÄâÊã©ÈÖçÁΩÆ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboExamConfigs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteConfig">
           <property name="text">
            <string>Âà†Èô§</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_EC_ConfigSave">
         <item>
          <widget class="QLabel" name="lbl_ConfigName">
           <property name="text">
            <string>ÈÖçÁΩÆÂêçÁß∞:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editConfigName">
           <property name="placeholderText">
            <string>ËæìÂÖ•ÈÖçÁΩÆÂêçÁß∞...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveConfig">
           <property name="text">
            <string>‰øùÂ≠òÈÖçÁΩÆ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_EC_Sep">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_ExamInputs">
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="l1">
           <property name="text">
            <string>È¢òÂûã</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="l2">
           <property name="text">
            <string>ÂèØÁî®</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="l3">
           <property name="text">
            <string>È¢òÊï∞</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="l4">
           <property name="text">
            <string>ÂàÜÂÄº</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="l_s">
           <property name="text">
            <string>ÂçïÈÄâ</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbl_Avail_Single">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="edit_Count_Single">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="edit_Score_Single">
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="l_m">
           <property name="text">
            <string>Â§öÈÄâ</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lbl_Avail_Multi">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="edit_Count_Multi">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="edit_Score_Multi">
           <property name="text">
            <string>2.0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="l_j">
           <property name="text">
            <string>Âà§Êñ≠</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lbl_Avail_Judge">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="edit_Count_Judge">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="edit_Score_Judge">
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="l_f">
           <property name="text">
            <string>Â°´Á©∫</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lbl_Avail_Fill">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="edit_Count_Fill">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="edit_Score_Fill">
           <property name="text">
            <string>2.0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="l_time">
           <property name="text">
            <string>ÈôêÊó∂ (ÂàÜÈíü):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="edit_ExamTime">
           <property name="text">
            <string>60</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer_EC_Middle">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnStartExam">
         <property name="text">
          <string>üöÄ ÂºÄÂßãËÄÉËØï</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
